app/models/end_user.rb

```diff
@@ -1,8 +1,5 @@
 # Subclass of User that can be associated with Roles
```

Comment:
This comment should be updated to something to something a bit more general, like "for regular users".